Successfully updated:
Created Frame node gdFpQaTsf
Created Frame node TS51kTpoF
Created Text node Epm0qZ7Om
Created Text node Emq2LCsBG
Created Frame node jRefPBpVC
Updated attributes for node gdFpQaTsf
Updated attributes for node TS51kTpoF
Updated text for node Epm0qZ7Om
Updated attributes for node Epm0qZ7Om
Updated text for node Emq2LCsBG
Updated attributes for node Emq2LCsBG
Updated attributes for node jRefPBpVC
Reordered node TS51kTpoF within parent gdFpQaTsf to index 0

XML Changes:
Index: node.xml
===================================================================
--- node.xml	Before
+++ node.xml	After
@@ -2977,20 +2977,73 @@
     position="absolute"
     width="fit-content"
     height="fit-content"
     top="0px"
     left="0px"
     font="Inter"
 >
   Body content goes here
 </BodyContentGoesHere>
 <Something
     nodeId="so_W61aYl"
     position="absolute"
     width="fit-content"
     height="fit-content"
     top="0px"
     left="0px"
     font="Inter"
 >
   something
 </Something>
+<Stack
+    nodeId="gdFpQaTsf"
+    position="absolute"
+    width="400px"
+    height="300px"
+    top="0px"
+    left="0px"
+    <!-- background color string or project color style path (if starts with /) -->
+    backgroundColor="rgb(240, 240, 240)"
+    layout="stack"
+    gap="16px"
+    padding="20px"
+    stackDirection="vertical"
+    stackDistribution="center"
+    stackAlignment="center"
+>
+  <Frame
+      nodeId="TS51kTpoF"
+      width="100%"
+      height="60px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(100, 150, 200)"
+      borderRadius="8px"
+  >
+    <HeaderText
+        nodeId="Epm0qZ7Om"
+        position="absolute"
+        width="fit-content"
+        height="fit-content"
+        top="0px"
+        left="0px"
+        font="Inter"
+    >
+      Header Text
+    </HeaderText>
+  </Frame>
+  <BodyContentGoesHere
+      nodeId="Emq2LCsBG"
+      width="fit-content"
+      height="fit-content"
+      font="Inter"
+  >
+    Body content goes here
+  </BodyContentGoesHere>
+  <Frame
+      nodeId="jRefPBpVC"
+      width="100%"
+      height="100px"
+      <!-- background color string or project color style path (if starts with /) -->
+      backgroundColor="rgb(255, 255, 255)"
+      borderRadius="4px"
+   />
+</Stack>
